### PR TITLE
[Misc] Refactor the instruction OpCode.

### PR DESCRIPTION
**Pull request by q82419**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by q82419** [comment]
Comment: performance should be tested with this change. But OpCode refactoring still take times, will continue after improvement of exception-handling proposal.

File: include/common/enum.inc [hunk 1]
```diff
@@ -85,605 +85,628 @@ A(Sec_AOT, "aot section")
...
-O(I32__atomic__rmw16__cmpxchg_u, 0xFE4B, "i32.atomic.rmw16.cmpxchg_u")
-O(I64__atomic__rmw8__cmpxchg_u, 0xFE4C, "i64.atomic.rmw8.cmpxchg_u")
-O(I64__atomic__rmw16__cmpxchg_u, 0xFE4D, "i64.atomic.rmw16.cmpxchg_u")
-O(I64__atomic__rmw32__cmpxchg_u, 0xFE4E, "i64.atomic.rmw32.cmpxchg_u")
+O(Br_on_null, "br_on_null", 0xD5)
+O(Br_on_non_null, "br_on_non_null", 0xD6)
+// 0xD7 ~ 0xFA: Reserved
+
+// 0xFB prefix - GC Instructions
+OFB(Struct__new, "struct.new", 0xFB, 0)
```
Comment (hydai) [inline include/common/enum.inc:312]: Should we also use the hex encoding for the last parameter? I am not sure why the third one uses `hex`, but the last one uses `dec`.

**Review by hydai — CHANGES_REQUESTED** [review]

File: include/common/enum.inc (hunk 1 above)
Comment (q82419) [reply include/common/enum.inc:312]: Refer to the instruction binary:
https://webassembly.github.io/spec/core/binary/instructions.html#vector-instructions

The second OpCode is a `u32` integer, encoded with LEB-128.
Therefore we use the dec here to present the real decoded `u32` number.

For example, the `f32x4.pmax` instruction, which OpCode is `0xFD 235:u32`, which binary format is `0xFD 0xEB 0x01`.

**Review by hydai — APPROVED** [review]

**Pull request merged**